Fix the CI: `.../golang/lint/golint expects import "golang.org/x/lint/golint"`

All new builds are failing for good code on:
```
package github.com/golang/lint/golint: code in directory /home/travis/gopath/src/github.com/grumpyhome/grumpy/grumpy-runtime-src/build/src/github.com/golang/lint/golint expects import "golang.org/x/lint/golint"
```

Travis logs context:
```bash
The command "pytest" exited with 0.
$ cd ../grumpy-runtime-src
The command "cd ../grumpy-runtime-src" exited with 0.
$ make gofmt lint && make -j2 test
find: `compiler': No such file or directory
find: `compiler': No such file or directory
find: `compiler': No such file or directory
package github.com/golang/lint/golint: code in directory /home/travis/gopath/src/github.com/grumpyhome/grumpy/grumpy-runtime-src/build/src/github.com/golang/lint/golint expects import "golang.org/x/lint/golint"
make: *** [build/bin/golint] Error 1
The command "make gofmt lint && make -j2 test" exited with 2.
Done. Your build exited with 1.
```

--- a/grumpy-runtime-src/Makefile
+++ b/grumpy-runtime-src/Makefile
@@ -237,7 +237,7 @@ gofmt: build/gofmt.diff
 	@if [ -s $< ]; then echo 'gofmt found errors, run: gofmt -w $(ROOT_DIR)/runtime/*.go'; false; fi
 
 $(GOLINT_BIN):
-	@go get -u github.com/golang/lint/golint
+	@go get -u golang.org/x/lint/golint
 
 golint: $(GOLINT_BIN)
 	@$(GOLINT_BIN) -set_exit_status runtime

